diff --git a/invesalius/inv_paths.py b/invesalius/inv_paths.py
index 544b82d2..5e53dbaf 100644
--- a/invesalius/inv_paths.py
+++ b/invesalius/inv_paths.py
@@ -2,6 +2,7 @@ import os
 import pathlib
 import shutil
 import sys
+import tempfile
 
 USER_DIR = pathlib.Path().home()
 CONF_DIR = pathlib.Path(os.environ.get("XDG_CONFIG_HOME", USER_DIR.joinpath(".config")))
@@ -9,6 +10,7 @@ USER_INV_DIR = CONF_DIR.joinpath("invesalius")
 USER_PRESET_DIR = USER_INV_DIR.joinpath("presets")
 USER_LOG_DIR = USER_INV_DIR.joinpath("logs")
 USER_RAYCASTING_PRESETS_DIRECTORY = USER_PRESET_DIR.joinpath("raycasting")
+TEMP_DIR = tempfile.gettempdir()
 
 OLD_USER_INV_DIR = USER_DIR.joinpath(".invesalius")
 OLD_USER_PRESET_DIR = OLD_USER_INV_DIR.joinpath("presets")
diff --git a/invesalius/session.py b/invesalius/session.py
index b1bcdbee..18617584 100644
--- a/invesalius/session.py
+++ b/invesalius/session.py
@@ -88,11 +88,6 @@ class Session(with_metaclass(Singleton, object)):
 
     def CreateItens(self):
         import invesalius.constants as const
-        homedir = inv_paths.USER_DIR
-        tempdir = os.path.join(inv_paths.USER_DIR, u".invesalius", u"temp")
-        if not os.path.isdir(tempdir):
-            os.makedirs(tempdir)
-
         self._values = collections.defaultdict(dict, {
             'session': {
                 'mode': const.MODE_RP,
@@ -111,7 +106,7 @@ class Session(with_metaclass(Singleton, object)):
 
             'paths': {
                 'homedir': str(inv_paths.USER_DIR),
-                'tempdir': os.path.join(homedir, u".invesalius", u"temp"),
+                'tempdir': str(inv_paths.TEMP_DIR),
                 'last_dicom_folder': '',
             },
         })
