diff --git a/invesalius/data/imagedata_utils.py b/invesalius/data/imagedata_utils.py
index 75fa77c9..8d84b506 100644
--- a/invesalius/data/imagedata_utils.py
+++ b/invesalius/data/imagedata_utils.py
@@ -101,7 +101,7 @@ def ResampleImage2D(imagedata, px=None, py=None, resolution_percentage = None,
     resample.SetInputData(imagedata)
     resample.SetAxisMagnificationFactor(0, factor_x)
     resample.SetAxisMagnificationFactor(1, factor_y)
-    resample.SetOutputSpacing(spacing[0] * factor_x, spacing[1] * factor_y, spacing[2])
+    #  resample.SetOutputSpacing(spacing[0] * factor_x, spacing[1] * factor_y, spacing[2])
     if (update_progress):
         message = _("Generating multiplanar visualization...")
         resample.AddObserver("ProgressEvent", lambda obj,
