diff --git a/invesalius/constants.py b/invesalius/constants.py
index 36fa3ac6..ad169cc5 100644
--- a/invesalius/constants.py
+++ b/invesalius/constants.py
@@ -27,7 +27,7 @@ from invesalius import utils
 from invesalius import inv_paths
 
 #from invesalius.project import Project
-INVESALIUS_VERSION = "3.1.1"
+INVESALIUS_VERSION = "3.1.99993"
 
 INVESALIUS_ACTUAL_FORMAT_VERSION = 1.1
 
diff --git a/invesalius/gui/dialogs.py b/invesalius/gui/dialogs.py
index eb3e8150..99dff58b 100644
--- a/invesalius/gui/dialogs.py
+++ b/invesalius/gui/dialogs.py
@@ -1240,7 +1240,7 @@ def ShowAboutDialog(parent):
 
     info = AboutDialogInfo()
     info.Name = "InVesalius"
-    info.Version = "3.1.1"
+    info.Version = const.INVESALIUS_VERSION
     info.Copyright = _("(c) 2007-2019 Center for Information Technology Renato Archer - CTI")
     info.Description = wordwrap(_("InVesalius is a medical imaging program for 3D reconstruction. It uses a sequence of 2D DICOM image files acquired with CT or MRI scanners. InVesalius allows exporting 3D volumes or surfaces as mesh files for creating physical models of a patient's anatomy using additive manufacturing (3D printing) technologies. The software is developed by Center for Information Technology Renato Archer (CTI), National Council for Scientific and Technological Development (CNPq) and the Brazilian Ministry of Health.\n\n InVesalius must be used only for research. The Center for Information Technology Renato Archer is not responsible for damages caused by the use of this software.\n\n Contact: invesalius@cti.gov.br"), 350, wx.ClientDC(parent))
 
@@ -1300,6 +1300,10 @@ def ShowAboutDialog(parent):
 
     info.Artists = [u"Otavio Henrique Junqueira Amorim"]
 
+    icon = wx.Icon()
+    icon.LoadFile(str(inv_paths.ICON_DIR.joinpath('invesalius.ico')))
+    info.Icon = icon
+
     # Then we call AboutBox providing its info object
     AboutBox(info)
 
